base_tier_validation/models/tier_review.py

```diff
@@ -77,5 +78,10 @@ def _compute_reviewer_ids(self):
         for rec in self:
             rec.reviewer_ids = rec._get_reviewers()
 
+    @api.depends("reviewer_ids")
+    def _compute_todo_by(self):
+        for rec in self:
+            rec.todo_by = ", ".join(rec.reviewer_ids.mapped("display_name"))
```

Comment:
I'm not sure I like this, if you make tiers by groups, the list can be massive. Some ideas:
1. You could show the group name if it is a tier by group.
2. You can truncate the list if too long. E.g: "James Jamison, Daniel Davidson, Jacks Jackson (and 12 more .).

Reply:
Agree with option 2, I will revise it a bit for that. Thanks for your review.

Reply:
![image](https://user-images.githubusercontent.com/1973598/108344172-65a01d00-720f-11eb-9a07-105f9269575c.png)

Reply:
Done!

Reply:
:+1:

Reply:
That is great, thanks!